openedge-plugin/src/main/java/org/sonar/plugins/openedge/foundation/OpenEdgeSettings.java

```diff
@@ -294,6 +296,18 @@ public RefactorSession getProparseSession() {
     return proparseSession;
   }
 
+  /**
+   * Force usage of sonar.sourceEncoding property as SonarLint doesn't set correctly encoding
+   */
+  private Charset encoding() {
+      String encoding = settings.getString(CoreProperties.ENCODING_PROPERTY);
+      if (Strings.isNullOrEmpty(encoding)) {
+        return Charset.defaultCharset();
+      } else {
+        return Charset.forName(encoding.trim());
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') NullPointerException might be thrown as 'encoding' is nullable here [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=squid%3AS2259)